### ISSUE TITLE
Use the events returned by the /launch endpoints

### ISSUE THREAD

**Issue by gastaldi** [fabric8-launcher/ngx-launcher#295]
This is a followup to https://github.com/fabric8-launcher/launcher-planning/issues/167 and the backend change https://github.com/fabric8-launcher/launcher-backend/pull/457.

Since OSIO /launch endpoints differ in the number of events fired compared to /launcher/launch, the */launch endpoints now return (together with the WebSocket URL) the events that the websocket may receive. This should used preferably instead of assuming the initial handshake content received when a Websocket connection is established.

**Comment by edewit** [comment]
@gastaldi atm the first message on the websocket does this, it returns the status events that the client has to wait for first and subsequently sends the events them self

**Comment by gastaldi** [comment]
@edewit yes, but this first message always returns the same content and that is not correct in `/osio/import` for example, which triggers just a subset of that

**Comment by edewit** [comment]
so can't we "just" fix that instead of using a different method?

**Comment by gastaldi** [comment]
@edewit how would you fix that without introducing a different status endpoint?
